### PR TITLE
Deprecate unused/moved methods from Browser Context

### PR DESCRIPTION
**Pull request by miguelcleverti**
This PR removes some of the methods in the Browser Context that were not being used by any Behat test.

### CODE REVIEW

**Comment by andrerom** [comment]
It looks like you are doing several things here

**Comment by miguelcleverti** [comment]
@andrerom besides what is in the description there is just one small cs fix, I made so ezrobot would be happy :)

**Comment by miguelcleverti** [comment]
Added to this PR also the removal of method that have been move into PlatformUI context ezsystems/PlatformUIBundle/pull/535

**Comment by miguelcleverti** [comment]
ping @bdunogier

File: Context/Browser/SubContext/CommonActions.php [hunk 1]
```diff
@@ -42,9 +42,14 @@ public function visit( $path )
      * Waits maximum 3 seconds, tries every 200ms
      *
      * @link http://docs.behat.org/en/v2.5/cookbook/using_spin_functions.html
+     * @deprecated deprecated since version 7.0.0
      */
     public function iCanSee( $id )
     {
+        trigger_error(
+            "iCanSee is deprecated since v7.0.0 and will be removed in the future",
```
Comment (bdunogier) [inline Context/Browser/SubContext/CommonActions.php]: Since `6.x.x` (the next minor release), and it will be removed in `7.x`.

**Comment by bdunogier** [comment]
+1, but needs to be merged _after_ https://github.com/ezsystems/PlatformUIBundle/pull/535 has been merged;

**Pull request merged**